build.sbt

```diff
@@ -4,7 +4,7 @@ name:= "content-api-firehose-client"
 organization := "com.gu"
 scalaVersion := "2.12.17"
 crossScalaVersions := Seq(scalaVersion.value, "2.13.10")
-scalacOptions ++= Seq("-feature", "-deprecation", "-unchecked", "-release:8", "-Xfatal-warnings")
```

Comment:
Is it completely safe to remove the `-release:8` option?

Reply:
It was complaining about it when doing the Scala 2.13 cross-build - looks like a subtle syntax change. According to the docs, the default is always v8 so I believe that it is safe to remove